recipes/geojsonio/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.0.2" %}
+
+package:
+    name: geojsonio.py
+    version: {{ version }}
+
+source:
+    fn: geojsonio-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/g/geojsonio/geojsonio-{{ version }}.tar.gz
+    md5: e75b0ec0786d456dc630f93874106bfd
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+    entry_points:
+        - geojsonio = geojsonio:main
+
```

Comment:
Most pure python packages recipes, that are on PyPI, will start from `conda-skeleton`, and `conda-skeleton` did add that space in this case. Not sure if that is always true, but if it is we might want to enforce that to keep consistency. (Although I don't have any strong feelings about it.)